mappings/net/minecraft/client/gl/JsonGlProgram.mapping

```diff
@@ -1,4 +1,4 @@
-CLASS dfn net/minecraft/client/gl/GlProgram
+CLASS dfn net/minecraft/client/gl/JsonGlProgram
```

Comment:
Is this warranted?

Reply:
It implements `GlProgram`. Otherwise, how should the interface be named?

Reply:
Ah, right.